docs/labs/4-ACM.md

```diff
@@ -362,9 +362,23 @@ you can call ```getWidth()``` or ```getHeight()```.
 You can use this in combination with a ```setLocation(x, y)```
 that will change the object's location if you want the object to be placed in a particular area.
 
+There are different types of Images that can be imported. Some include:
+
+1. Portable Network Graphics (.png)
+2. Joint Photographic Experts Group (.jpg)
+3. Graphics Interchange Format (.gif)
+
+What is the point of all these different file types?
+
+1. PNG - is compatible as a clear background, so if you can blend with your design use a png.
+2. JPG - is usually the best quality image, so use it if you do not need to blend images.
+3. GIF - is nice to use as an easy animation. Insert a gif and the image will be animated.
```

Comment:
Can we just combine these two parts, so that perhaps after the png we can just give a description?
So something like.

```
1. Portable Network Graphics (.png) - is compatible etc...
```
Once you do that re-request a review and then we'll done one more round